infrastructure/nomad/playbooks/deploy.yml

```diff
@@ -302,21 +304,34 @@
               {% endif %}
             {% endif %}
 
-            {% if artifact.boot_key | default(false) %}
+            {% if artifact.boot_key is defined %}
```

Comment:
The file from which a Geth node reads the key is called `boot.key`.

Reply:
```
--nodekey value                                                        ($GETH_NODEKEY)
          P2P node key file
```

This is from the geth command. In geth context, there is nothing called boot key. We call it boot key because we initially had it only for bootnode. But calling it boot key doesnt make any sense to me. We can't change the name of the file?

Reply:
Done.